README.md

```diff
@@ -210,6 +210,7 @@ API | Description | Auth | HTTPS | CORS |
 ### Business
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [API Gratis](http://apigratis.com.br/) | Services and API free | No | No | Unknown |
```

Comment:
It is not allowed to repeat the APIs in other sections, it must only be in 1 section.

Remove this, please:

```suggestion
```